src/nft/components/explore/CarouselCard.tsx

```diff
@@ -12,14 +12,20 @@ const CarouselCardContainer = styled.div`
   flex-direction: column;
   gap: 20px;
   background-color: ${({ theme }) => theme.backgroundSurface};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
   border-radius: 20px;
-  outline: 1px solid ${({ theme }) => theme.backgroundOutline};
+  overflow: hidden;
+  height: 100%;
+`
+const CarouselCardBorder = styled.div`
   width: 100%;
+  position: relative;
+  border-radius: 22px;
   cursor: pointer;
-  overflow: hidden;
+  border: 2px solid transparent;
 
   :hover {
-    outline: 3px solid ${({ theme }) => theme.backgroundOutline};
+    border: 2px solid ${({ theme }) => theme.backgroundOutline};
     box-shadow: ${({ theme }) => theme.deepShadow};
```

Comment:
would also be good to add a transition on this, but see: https://tobiasahlin.com/blog/how-to-animate-box-shadow/